wp-content/mu-plugins/pantheon/pantheon-login-form-mods.php

```diff
@@ -0,0 +1,55 @@
+<?php
```

Comment:
It would be nice if this plugin adhered to the [WordPress Coding Standards](https://codex.wordpress.org/WordPress_Coding_Standards)